vaadin-components-gwt/src/main/webapp/vaadin-grid/vaadin-grid.html

```diff
@@ -779,19 +779,26 @@
           this.contentDocument.defaultView.addEventListener('resize', function(e) {
             _this._grid.redraw();
           });
-          _this._grid.redraw();
+
+          _this.then(function() {
```

Comment:
Forgot this comment here, there are more than one nested level here, it's more appropriate in the code below.